src/core/time.d

```diff
@@ -2645,9 +2650,10 @@ extern(C) void _d_initMonoTime()
         {
             foreach(i, typeStr; __traits(allMembers, ClockType))
             {
+                import core.internal.abort : abort;
                 // ensure we are only writing immutable data once
                 if(tps[i] != 0)
-                    assert(0, "_d_initMonoTime should only be called once!");
+                    abort("_d_initMonoTime should only be called once!");
```

Comment:
If it's really an environmental issue, we should just be throwing Errors IMHO. There might be cases where you wouldn't want to do that, but they'd be pretty atypical, I would think. Really, it looks to me like `abort` is just `assert(0, msg)` that still prints a message when it fails in release mode.